src/main/kotlin/no/nav/familie/ef/personhendelse/client/SakClient.kt

```diff
@@ -22,4 +22,11 @@ class SakClient(
         val response = postForEntity<Ressurs<Boolean>>(uriComponentsBuilder.build().toUri(), personidenter)
         return response.data ?: error("Kall mot ef-sak feilet. Status=${response.status} - ${response.melding}")
     }
+
+    fun hentFnrForAlleBarn(personIdent: String): List<String> {
+        val uriComponentsBuilder = UriComponentsBuilder.fromUri(uri)
+            .pathSegment("/api/behandling/barn/$personIdent")
+        val response = getForEntity<Ressurs<List<String>>>(uriComponentsBuilder.build().toUri())
+        return response.data ?: error("Kall mot ef-sak feilet. Status=${response.status} - ${response.melding}")
```

Comment:
Hvis man skulle få noe annet enn 200, så kaster getForEntity feil, sånn att man kan bruke `getDataOrThrow ` her,
```suggestion
        return response. getDataOrThrow()
```